cli/clitest/clitest.go

```diff
@@ -46,6 +48,29 @@ func CreateTemplateVersionSource(t *testing.T, responses *echo.Responses) string
 	return directory
 }
 
+// GenTemplateVersion returns a unique bundle that can be used to create
+// a template version source.
+func GenTemplateVersion() *echo.Responses {
+	return &echo.Responses{
+		Parse: []*proto.Parse_Response{
+			{
+				Type: &proto.Parse_Response_Log{
+					Log: &proto.Log{
+						Output: uuid.NewString(),
+					},
+				},
+			},
+
+			{
+				Type: &proto.Parse_Response_Complete{
+					Complete: &proto.Parse_Complete{},
+				},
+			},
+		},
+		Provision: echo.ProvisionComplete,
+	}
+}
```

Comment:
This isn't _exactly_ generalizable, so I might keep this in your test.

This is typically used to pass agent tokens, create fake resources, apps, etc.